packages/tc-ui/src/lib/components/structure.jsx

```diff
@@ -139,11 +139,21 @@ const MetaProperties = ({ data, isCreate }) => {
 	);
 };
 
-const LinkToRecord = ({ id, type, value: { name, code } }) => (
-	<a id={id} href={`/${type}/${encodeURIComponent(code)}`}>
-		{name || code}
-	</a>
-);
+const LinkToRecord = ({
+	id,
+	type,
+	value: { name, code },
+	isRunbooksSystemLink,
+}) => {
+	const href = isRunbooksSystemLink
```

Comment:
It's important to remember that treecreeper shouldn't know anything about biz ops, runbooks etc, so referring to runbooks in the codebase is a no no. Think more in terms of naming things something like 'alternativeLinkFormat`. Once you restrict yourself to not allowing any mention of runbooks, or specifying the actual link format in tc-ui code, what approaches are left that you can take?

Reply:
I have replaced it with `alternativeHostname` and we send this from runbooks as `https://runbooks.in.ft.com` if type is System instead of a boolean value